### ISSUE TITLE
[Feature request] Labeless syncs function/procedure list from IDA to OllyDbg/other debuggers

### ISSUE THREAD

**Issue by trietptm** [a1ext/labeless#10]
I see the function analysis of IDA is much better than OllyDbg so that I want Labeless not only syncs function names from IDA but also marks the functions which OllyDbg doesn't recognize as functions/procedures.

Here, the label of the function is synced:
![screenshot - 7_14_2016 5_20_44 pm](https://cloud.githubusercontent.com/assets/526959/16836578/6c3c2496-49e8-11e6-9bda-b1a3f7b70b5c.png)

Here, the procedure list doesn't include the function synced from IDA:
![screenshot - 7_14_2016 5_21_34 pm](https://cloud.githubusercontent.com/assets/526959/16836417/98f9e92e-49e7-11e6-974e-7e1c1a0e8c84.png)

**Comment by a1ext** [comment]
Hm... I don't know is this possible to tell to Olly about new procedure (I think Olly makes procedures list based on initial internal Olly's analysis) using Olly PDK. Maybe we can force to reanalyze some memory region...

**Comment by mrexodia** [comment]
For x64dbg this is possible with `Script::Function::Add`

**Comment by a1ext** [comment]
@mrexodia Thanks

**Comment by a1ext** [comment]
@mrexodia Hi, does x64dbg have a possibility to specify a number of arguments for a function and give them all size/type (on case of enum, struct, etc.)? Also, I may provide to the debugger some information about known APIs prototypes to help the debugger to show arguments in the stack on reaching 'call API'

**Comment by mrexodia** [comment]
No, this is currently not supported. You can make comments though to
display them. There is also an arguments widget but there is no support for
names or types yet.

**Comment by mrexodia** [comment]
There has been progress on this. You can now define functions but enum
types are not yet supported and you cannot do much concrete with the
functions currently. See https://github.com/x64dbg/x64dbg/issues/1108 for
more details.
